apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/pensjonforvalter/PensjonforvalterClient.java

```diff
@@ -83,16 +83,16 @@ public void gjenopprett(RsDollyUtvidetBestilling bestilling, DollyPerson dollyPe
 
         StringBuilder status = new StringBuilder();
 
+        if (!dollyPerson.isOpprettetIPDL()) {
+            progress.setPensjonforvalterStatus(bestilling.getEnvironments().stream()
```

Comment:
jeg tror den mangler 
PENSJON_FORVALTER + "#"
for status

Reply:
Rettet, takk bra sett! 👍